lang/fr.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+$lang['title'] = 'TecArt Ticket System';
+
+$lang['foot_note'] = 'Ticket-Web Interface &copy; by TecArt GmbH';
+
+$lang['custom_number']  = 'N ° de client';
+$lang['password']       = 'Mot de passe';
+$lang['login']          = 'S\'identifier';
+$lang['headline_standard'] = 'Ticket-Web Interface';
+$lang['headline']       = 'TecArt Ticket-Web Interface';
+$lang['logout']         = 'Connectez - Out';
+$lang['view']           = 'Vue';
+
+$lang['show_current']       = 'Billets actuels';
```

Comment:
Same as in #11 - is "billets" the correct word for incident tickets?

Reply:
"billets" is common french too